sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2ScanRelationPushDown.scala

```diff
@@ -93,8 +93,12 @@ object V2ScanRelationPushDown extends Rule[LogicalPlan] with PredicateHelper {
                     agg
                 }
               }
+              aggregates = DataSourceStrategy.normalizeExprs(aggregates, sHolder.relation.output)
```

Comment:
`val normalizedAggregates`? So you can keep `val aggregates` above.

Reply:
Done. Thanks!